specification/network/resource-manager/Microsoft.Network/stable/2019-08-01/examples/P2SVpnGatewayGetConnectionHealthDetailed.json

```diff
@@ -0,0 +1,23 @@
+{ 
+    "parameters": { 
+        "subscriptionId": "subid", 
+        "resourceGroupName": "p2s-vpn-gateway-test", 
+        "gatewayName": "p2svpngateway", 
+        "api-version": "2019-06-01", 
+        "request": { 
+            "p2sVpnConnectionHealthFilters": [ 
+                { 
+                    "vpnUserNames":[                         
+                        "vpnUser1", 
+                        "vpnUser2"                         
+                    ]                               
+                } 
+            ], 
+            "outputBlobSasUrl": "https://blobcortextesturl.blob.core.windows.net/folderforconfig/p2sconnectionhealths?sp=rw&se=2018-01-10T03%3A42%3A04Z&sv=2017-04-17&sig=WvXrT5bDmDFfgHs%2Brz%2BjAu123eRCNE9BO0eQYcPDT7pY%3D&sr=b" 
+        } 
+    }, 
+    "responses": { 
+        "200": {}, 
```

Comment:
`200` should return `P2SVpnConnectionHealth` according to specs

Reply:
Done.